pkg/migrate/migrate.go

```diff
@@ -541,6 +584,10 @@ func validateStorageClasses(ctx context.Context, w *log.Logger, clientset k8scli
 		}
 		if sc.Name == destSCName {
 			destScFound = true
+
+			if sc.Provisioner == openEBSLocalProvisioner {
+				isDestScLocalVolumeProvisioner = true
+			}
```

Comment:
I don't like special casing this provisioner
I'm fine with adding logic for it, but I'd rather have that logic work regardless of what the provisioner name is

Reply:
once again lets not change anything now for other provisioners and instead follow up with a change to unify the code